src/helpers/proofs/fetchCentralizedProof.ts

```diff
@@ -24,15 +24,7 @@ export default async function (
 }
 
 async function sendRequest(id: string, proverAddress: string) {
```

Comment:
If you'll return the promise, the function shouldn't be async

Reply:
The data is returned

Reply:
Fixed